content/05.proteolysis.md

```diff
@@ -8,13 +8,65 @@ Peptides that are too short are less likely to uniquely match to a single protei
+9. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
+10. Chymotrypsin:
+Chymotrypsin or Chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas and works at an optimum pH ranging from 7.8 to 8.0. 
+It cleaves at the C-terminus of hydrphobic amino acids Phe, Trp, Tyr and barely Met and Leu residues.
+Since the transmembrane region of membrance proteins lacks tryptic cleavage sites, this enzyme works well with membrane proteins having more hydrophobic residues [@PMID: 24870543; @PMID:24696503; @PMID:27123950].
+Also, the chymotryptic peptides generated after proteolysis will cover the proteome space orthogonal to that of tryptic peptides both in a quantitative and qualitative manner [@PMID:24290761; @PMID:22669647; @PMID:24696503]
+11. Arg-C:
+Clostripain, commonly known as Arg-C, is a cysteine protease obtained from _Clostridium histolyticum_.
+It hydrolyse mostly the C-terminal Arg residues and sometimes Lys residues, but with less efficiency.
+The peptides generated are generally longer than that of tryptic peptides.
+Arg-C is often used with other proteases for improving qualitative proteome data and also for investigating PTMs [@PMID:25823410]. 
+12. LysargiNase:
+LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family. 
+It is a thermophilic protease derived from _Methanosarcina acetivorans_ that specifically cleaves at the N-terminus of Lys and Arg residues [@PMID:25419962].
+Hence, it enabled in the determination of C-terminal peptides, which otherwise was not possible using trypsin.
+In addition, it also favours the hydr0lysis of modified amino acids such as dimethylated or methylated arginine and lysine [@PMID:25419962].
+13. Lys-N:
+Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from _Grifola frondosa_. 
+It cleaves at N-terminal of Lys and has an optimal activity at pH 9.0.
+Unlike trypsin, Lys-N is more resistant to denaturing agents and can be heated up to 70 °C.
+Reports suggests that the peptides generated after Lys-N digestion produces more of c-type ions in a ETD-based mass spectrometer. 
+Hence this can be used for analysing PTMs, identification of C-terminal peptides and also for de novo sequencing strategies [@PMID:18425140; @PMID:20953479].
+14. Pepsin:
+Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcrine pancreas. 
+This was the first protease that was identified and crystallized and was awarded the Nobel prize.
+Pepsin works at an optimum pH from 1-4 and, in a pH depended manner it specifically cleaves Trp, Phe, Tyr and  Leu [@PMID:25823410]. 
+Since it possess high enzyme activity and broad specificity at lower pH, it is preferred over other proteases for MS-based disulphide mapping [@PMID:12476442; @PMID:24980484].
+
```

Comment:
```suggestion
Pepsin is also used extensively for structural mass spectrometry studies with hydrogen-deuterium exchange (HDX) because the rate of back exchange of the amide deuteron is minimized at low pH [@DOI:10.1021/ac902477u; @DOI:10.1002/mas.21565].  
```